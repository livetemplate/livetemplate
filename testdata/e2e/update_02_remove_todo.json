{"1":"Task Manager","2":"8","9":"2","10":"1","11":"1","13":"50","31":"2023-01-01 10:30:00","32":"session-12345"}