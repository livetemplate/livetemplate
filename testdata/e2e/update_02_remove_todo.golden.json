{
  "1": "Task Manager",
  "2": "8",
  "3": "active",
  "4": "High Activity",
  "5": "2",
  "6": "1",
  "7": "1",
  "8": "50%",
  "9": [
    [
      "u",
      "completed",
      {
        "0": "1",
        "2": "#0",
        "3": "Learn Go templates",
        "5": "(Priority: High)"
      }
    ],
    [
      "u",
      "",
      {
        "0": "2",
        "2": "#1"
      }
    ]
  ],
  "10": "2023-01-01 10:30:00",
  "11": "session-12345"
}